teps/0091-trusted-resources.md

```diff
@@ -377,13 +386,44 @@ metadata:
   name: feature-flag
   namespace: tekton-pipelines
 data:
-  verification-policy: enforce
+  enable-trusted-resource: true
 ```
 
-`verification-policy`. (Optional, `enforce`, `warn` or `skip`, default `skip`):
- * `enforce`: Failing verification will mark the taskruns/pipelineruns as failed.
- * `warn`: Log warning but don't fail the taskruns/pipelineruns.
- * `skip`: Directly skip the verification.
+`enable-trusted-resource`. (Optional, `true` or `false`, default `false`):
+ * `true`: Failing verification will mark the taskruns/pipelineruns as failed.
```

Comment:
Warn throws a wrench into this. This might be another reason to just drop the feature flag entirely and just rely on policy existence. 🤔

Reply:
I was thinking about implementation details and thought having this might be easier. 

Without this, if we want to turn off this feature:
we need to set the no-match-policy to "allow", and make sure no policies are in the namespace. This may need some operations on the cluster. But if we have the feature flag, users can just turn off there without removing the policies. 

I think another difference between this work and policy-controller is that, trusted resources is just one feature of tekton pipeline. Policy controller is a project designed for images verification. Users will use it once they deploy the pods, and if they don't want to use they can remote the deployment.

Reply:
updated to remove this and rename the current feature flag to "no-match-policy"